src/elements/form/Control.ts

```diff
@@ -28,90 +28,103 @@ import { Helpers } from './../../utils/Helpers';
+                            <!--QuickNote-->
+                            <novo-quick-note *ngSwitchCase="'quick-note'" [formControlName]="control.key" [placeholder]="control.placeholder" [config]="control.config" (change)="modelChange($event)"></novo-quick-note>
+                             <!--ReadOnly-->
+                            <!--TODO - Handle rendering of different READONLY values-->
+                            <div *ngSwitchCase="'read-only'">{{ form.value[control.key] }}</div>
+                        </div>
                     </div>
-                    <!--TextArea-->
-                    <textarea *ngSwitchCase="'text-area'" [name]="control.key" [attr.id]="control.key" [placeholder]="control.placeholder" [formControlName]="control.key" (input)="resizeTextArea($event)" (input)="emitChange($event)" [maxlength]="control.maxlength"></textarea>
-                    <!--Editor-->
-                    <novo-editor *ngSwitchCase="'editor'" [name]="control.key" [formControlName]="control.key" (focus)="handleFocus($event)" (blur)="handleBlur($event)"></novo-editor>
-                    <!--HTML5 Select-->
-                    <select [id]="control.key" *ngSwitchCase="'native-select'" [formControlName]="control.key">
-                        <option *ngIf="control.placeholder" value="" disabled selected hidden>{{control.placeholder}}</option>
-                        <option *ngFor="let opt of control.options" [value]="opt.key">{{opt.value}}</option>
-                    </select>
-                    <!--File-->
-                    <novo-file-input *ngSwitchCase="'file'" [formControlName]="control.key" [id]="control.key" [name]="control.key" [placeholder]="control.placeholder" [multiple]="control.multiple"></novo-file-input>
-                    <!--Tiles-->
-                    <novo-tiles *ngSwitchCase="'tiles'" [options]="control.options" [formControlName]="control.key" (onChange)="modelChange($event)"></novo-tiles>
-                    <!--Picker-->
-                    <div class="novo-control-input-container" *ngSwitchCase="'picker'">
-                        <novo-picker [config]="control.config" [formControlName]="control.key" [placeholder]="control.placeholder" [appendToBody]="control.appendToBody" [parentScrollSelector]="control.parentScrollSelector" *ngIf="!control.multiple" (select)="modelChange($event);" (typing)="handleTyping($event)" (focus)="handleFocus($event)" (blur)="handleBlur($event)"></novo-picker>
-                        <chips [source]="control.config" [type]="control.config.type" [formControlName]="control.key" [placeholder]="control.placeholder" *ngIf="control.multiple" [closeOnSelect]="control.closeOnSelect" (changed)="modelChange($event)" (typing)="handleTyping($event)" (focus)="handleFocus($event)" (blur)="handleBlur($event)"></chips>
+                    <!--Error Message-->
+                    <div class="field-message" *ngIf="errors || showCount || control.description || !condensed">
```

Comment:
@jgodi i meant to set up field-message so that it appears at all times except for if not condensed. will fix that